src/main/flight/pid.h

```diff
@@ -149,9 +144,7 @@ typedef struct pidProfile_s {
     uint8_t horizon_tilt_expert_mode;       // OFF or ON
 
     // Betaflight PID controller parameters
-    uint8_t  antiGravityMode;             // type of anti gravity method
-    uint16_t itermThrottleThreshold;        // max allowed throttle delta before iterm accelerated in ms
-    uint16_t itermAcceleratorGain;          // Iterm Accelerator Gain when itermThrottlethreshold is hit
+    uint8_t itermAcceleratorGain;           // Iterm Accelerator Gain
```

Comment:
Can we call it simply `anti_gravity_gain`?

Reply:
> 

It has the effect of "boosting" your iterm value. So even though it isn't "boosting" the accumulated iterm, I'd still say that it is "boosting" by essentially giving a boost to your iterm.